docs/documentation/components/nav.html

```diff
@@ -53,20 +53,20 @@ <h2 class="subtitle">
 {% capture nav_example %}
 <nav class="nav">
   <div class="nav-left">
-    <a class="nav-item">
-      <img src="{{ site.url }}/images/bulma-logo.png" alt="Bulma logo">
+    <a class="nav-item" href="#">
+      <img src="{{ site.url }}/images/bulma-logo.png" alt="Bulma">
```

Comment:
Keep `alt="Bulma logo"`

Reply:
Why do we need to keep 'logo'?

Reply:
The `alt` attribute is supposed to describe what the image is, not what it says.

Reply:
and in our case, this is a link so the user needs to hear "Bulma link" not "Bulma logo link"the second one means if the user clicks on it it will open the bulma logo not redirect to the main page.

Reply:
You are mixing `alt` and `title` (I pass on aria attributes), `alt` stands for alertative content that should describe what is shown on the missing picture/media. The anchor should carry information on his path, img is described through his `alt` attribute.

Reply:
Actually the whole nav is deprecated so it doesn't matter.

Also, `fas fa-github` won't work since it needs `fab`.